new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorTwoDomains.java

```diff
@@ -0,0 +1,690 @@
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = domain1Uid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      domain1ManagedServerPodOriginalTimestampList.add(
+          getPodOriginalCreationTimestamp(managedServerPodName, domain1Namespace));
+
+      managedServerPodName = domain2Uid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      domain2ManagedServerPodOriginalTimestampList.add(
+          getPodOriginalCreationTimestamp(managedServerPodName, domain2Namespace));
+    }
+  }
+
+  /**
+   * Shutdown both domains and verify all the server pods were shutdown.
+   */
+  private void shutdownBothDomainsAndVerify() {
+
+    // shutdown both domains
+    logger.info("Shutting down both domains");
+    for (int i = 0; i < numberOfDomains; i++) {
+      shutdown(domainUids.get(i), domainNamespaces.get(i));
+    }
+
+    // verify all the pods were shutdown
+    logger.info("Verifying all server pods were shutdown for both domains");
+    for (int i = 0; i < numberOfDomains; i++) {
+      // check admin server pod was shutdown
+      checkPodDoesNotExist(domainUids.get(i) + "-" + ADMIN_SERVER_NAME_BASE,
+          domainUids.get(i), domainNamespaces.get(i));
+
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName = domainUids.get(i) + "-" + MANAGED_SERVER_NAME_BASE + j;
+        checkPodDoesNotExist(managedServerPodName, domainUids.get(i), domainNamespaces.get(i));
+      }
+    }
+
+    // check the scaled up managed servers in domain1 were shutdown
+    logger.info("Verifying the scaled up managed servers in domain1 were shutdown");
+    for (int i = replicaCount + 1; i <= replicasAfterScale; i++) {
+      String managedServerPodName = domain1Uid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      checkPodDoesNotExist(managedServerPodName, domain1Uid, domain1Namespace);
+    }
+
+  }
+
+  /**
+   * Get pod original creation timestamp.
+   * @param podName pod name to get the original creation timestamp
+   * @param namespace the namespace in which the pod exists
+   * @return the pod original creation timestamp
+   */
+  private String getPodOriginalCreationTimestamp(String podName, String namespace) {
+    return assertDoesNotThrow(() -> getPodCreationTimestamp(namespace, "", podName),
+        String.format("getPodCreationTimestamp failed with ApiException for pod %s in namespace %s",
+            podName, namespace));
+  }
+
+  /**
+   * Check pod exist, ready and service exists in the specified namespace.
+   *
+   * @param podName pod name to check
+   * @param domainUid the label the pod is decorated with
+   * @param namespace the namespace in which the pod exists
+   */
+  private void checkPodExistsReadyAndServiceExists(String podName, String domainUid, String namespace) {
+    logger.info("Checking that pod {0} exists in namespace {1}", podName, namespace);
+    checkPodExists(podName, domainUid, namespace);
```

Comment:
You don't need this check since podReady does this too.

Reply:
fixed